apps/official/components/basic-recommendations/WhatCanIDoPage.tsx

```diff
@@ -0,0 +1,198 @@
+// frameworks...
+import React, { ReactElement } from "react";
+import { useLocation } from "react-router-dom";
+import { useSelector } from "react-redux";
+import { makeStyles } from "@material-ui/core/styles";
+import { Grid, Typography, CircularProgress } from "@material-ui/core";
+
+// base app...
+import { formatUTCDate } from "../../../../src/lib/formatUTCDate";
+import { State } from "../../../../src/state";
+
+// apps/official...
+import NoContactsIcon from "../../static/images/no-contacts.svg";
+import DistanceIcon from "../../static/images/distance.svg";
+import HygieneIcon from "../../static/images/hand-washing.svg";
+import MaskIcon from "../../static/images/mask.svg";
+import VentilationIcon from "../../static/images/fresh-air.svg";
+import RegionalIcon from "../../static/images/checklist.svg";
+import { NavigationTitle } from "app-config/components/NavigationTitle";
+import { MeasureSection } from "./MeasureSection";
+import { CountyTeaser } from "./CountyTeaser";
+
+// FOR TRANSLATION -->
+const TITLE = "Wie kann ich mich verhalten?";
```

Comment:
```suggestion
const TITLE = "Wie sollte ich mich verhalten?";
```
Ich finde das klingt besser, können kann man ja vieles :)